h/templates/help.html.jinja2

```diff
@@ -3,9 +3,7 @@
 {% block meta %}
   <link href='//fonts.googleapis.com/css?family=Lato:400,300' rel='stylesheet' type='text/css'>
   {{ super() }}
-  <script>
-    function hypothesisConfig() { return {firstRun: true}; }
-  </script>
+  <script type="application/json" class="js-hypothesis-config">{"firstRun":"true"}</script>`
```

Comment:
The `true` here should be a boolean value rather than a string. I expect it works because the code just does `if (config.firstRun)`, but still better to use the right data type.